### PR TITLE
CIRC-1340 added implementation for fetchHoldingsRecords method

### PR DESCRIPTION
**Pull request by viktor-draban**
Link to [ticket](https://issues.folio.org/browse/CIRC-1340)

there was added implementation for fetchHoldingsRecords method

### CODE REVIEW

File: src/main/java/org/folio/circulation/infrastructure/storage/inventory/ItemRepository.java [hunk 1]
```diff
@@ -70,6 +71,7 @@
   private final boolean fetchLoanType;
 
   private static final String ITEMS_COLLECTION_PROPERTY_NAME = "items";
+  private static final String HOLDINGS_RECORDS_PREPERTY_NAME = "holdingsRecords";
```
Comment (sherzodn) [inline src/main/java/org/folio/circulation/infrastructure/storage/inventory/ItemRepository.java]: just correct like PROPERTY (it is PREPERTY)

File: src/main/java/org/folio/circulation/infrastructure/storage/inventory/ItemRepository.java [hunk 2]
```diff
@@ -70,6 +71,7 @@
   private final boolean fetchLoanType;
 
   private static final String ITEMS_COLLECTION_PROPERTY_NAME = "items";
+  private static final String HOLDINGS_RECORDS_COLLECTION_PREPERTY_NAME = "holdingsRecords";
```
Comment (sherzodn) [inline src/main/java/org/folio/circulation/infrastructure/storage/inventory/ItemRepository.java:74]: just correct like PROPERTY (not PREPERTY)

File: src/main/java/org/folio/circulation/infrastructure/storage/inventory/ItemRepository.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (viktor-draban) [reply src/main/java/org/folio/circulation/infrastructure/storage/inventory/ItemRepository.java]: done

File: src/main/java/org/folio/circulation/infrastructure/storage/inventory/ItemRepository.java [hunk 3]
```diff
@@ -385,6 +387,14 @@ private ItemRepository(Clients clients, LocationRepository locationRepository,
       .thenComposeAsync(this::fetchMaterialTypes);
   }
 
+  public CompletableFuture<Result<MultipleRecords<Holdings>>> findHoldingsByIds(
+    Collection<String> ids) {
+
+    return findWithMultipleCqlIndexValues(holdingsClient, HOLDINGS_RECORDS_PREPERTY_NAME, identity())
+      .findByIds(ids)
+      .thenApply(r -> r.map(multipleRecords -> multipleRecords.mapRecords(new HoldingsMapper()::toDomain)));
```
[the remark this answers is not in the document]
Comment (viktor-draban) [reply src/main/java/org/folio/circulation/infrastructure/storage/inventory/ItemRepository.java]: thanks. fixed

File: src/main/java/org/folio/circulation/infrastructure/storage/inventory/ItemRepository.java (hunk 1 above)
Comment (viktor-draban) [reply src/main/java/org/folio/circulation/infrastructure/storage/inventory/ItemRepository.java]: done

**Review by sherzodn — APPROVED** [review]

**Pull request merged**